ddht/v5_1/rpc_handlers.py

```diff
@@ -25,6 +26,10 @@ class PongResponse(TypedDict):
     packet_port: int
 
 
+class SendPingResponse(TypedDict):
+    request_id: HexStr
```

Comment:
> Request ID should be treated as `bytes` in all v51 contexts.

@pipermerriam Is it incorrect to use `HexStr` here? Using raw `bytes` causes problems with the json serialization.

Reply:
Totally fine to return it as a hex encoded value. :+1: